backend/app/api/statistics.py

```diff
@@ -19,11 +19,11 @@ class Statistics(Resource):
     @jwt_required()
     @get_request(Role.ADMIN)
     def get() -> Response:
-        parser = reqparse.RequestParser()
-        parser.add_argument('question_id', type=int, location='json', required=True)
-        parser.add_argument('min_value', type=int | str, location='json', required=False, default=None)
-        parser.add_argument('max_value', type=int | str, location='json', required=False, default=None)
-        parser.add_argument('step', type=int, location='json', required=True)
+        parser = GetRequestParser()
```

Comment:
Are you sure it works this way?
I thought type OR operator could be used only in type hinting
Please, check it